documentation/product/connectivity/topics/circuits-overview.md

```diff
@@ -0,0 +1,77 @@
+# Circuits Overview
+
+Manage and navigating the Circuits feature in Hyperview, offering users the ability to customize their data view, access detailed circuit information, and edit properties efficiently. With the ability to track changes, manage access control, and document connections, users can enhance their operational oversight and streamline their workflow.
+
+1.	Click "Circuits"
+
+```{image} /product/connectivity/media/circuits-overview/image1.jpg
+:class: border-black
+```
+
+2.	Edit which columns of data are visible on the Circuits overview page by Clicking the Columns Selector icon.
+
+```{image} /product/connectivity/media/circuits-overview/image2.jpg
+:class: border-black
+```
+
+3.	Toggle which Columns to display.
+
+```{image} /product/connectivity/media/circuits-overview/image3.jpg
+:class: border-black
+```
+
+4.	Select a Circuit, then click the "Circuit Info" button to view more information about the Circuit.
+
+```{image} /product/connectivity/media/circuits-overview/image4.jpg
+:class: border-black
+```
+
+```{image} /product/connectivity/media/circuits-overview/image5.jpg
+:class: border-black
+```
+
+5.	Click the details button to view and edit all Circuit properties and other information.
+
+```{image} /product/connectivity/media/circuits-overview/image6.jpg
+:class: border-black
+```
+
+6.	Connections contain editable custom properties, the same that you would find on any other Asset in Hyperview.
+
+```{image} /product/connectivity/media/circuits-overview/image7.jpg
+:class: border-black
+```
+
+7.	Documentation can also be added and linked to Connections.
```

Comment:
"Documentation" should probably be "Documents"

Reply:
Fixed